content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -462,49 +505,51 @@ manager for kubectl server-side apply is `kubectl`.
 kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
 ```
 
-## API Endpoint
-
-With the Server Side Apply feature enabled, the `PATCH` endpoint accepts the
-additional `application/apply-patch+yaml` content type. Users of Server Side
-Apply can send partially specified objects as YAML to this endpoint.  When
-applying a configuration, one should always include all the fields that they
-have an opinion about.
```

Comment:
I removed this because:
- Server-Side Apply is GA and the feature is always available
- `PATCH` is a verb, not an endpoint
- the new Serialization section of the page covers the detail about the media type and format.